src/main/java/com/flowable/flowableboot/utils/generateUmsTasks.java

```diff
@@ -0,0 +1,264 @@
+package com.flowable.flowableboot.utils;
+
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.UUID;
+
+public class generateUmsTasks {
```

Comment:
Some of these arrays should likely be enums, but I agree that it's not clear yet - for each, put a TODO: so that it's marked in the code based to figure it out later, and we can ctrl+f to find them easily.